mitosheet/src/mito/components/toolbar/HomeTabContents.tsx

```diff
@@ -146,6 +146,10 @@ export const HomeTabContents = (
                         }}/>
```

Comment:
In this file, the class `mito-toolbar-number-format-select` is reference but not used. I like using the `style` on the select in the way you do, though!